torch/onnx/utils.py

```diff
@@ -1240,7 +1238,7 @@ def get_ns_op_name_from_custom_op(symbolic_name):
                            alphanumerical characters"
                            .format(symbolic_name))
     ns, op_name = symbolic_name.split("::")
-    unaccepted_domain_names = ["onnx", "aten", "prim"]
+    unaccepted_domain_names = ["onnx", "aten"]
```

Comment:
We can leave this as-is for now to keep this PR focused on one thing, but I'd like to discuss: is there a good reason to disallow using "aten"? It seems like "foo" is actually treated implicitly as "aten::foo", so why do we disallow users from explicitly setting "aten"?

Reply:
This might be the legacy issue. Ideally we should not need this `unaccepted_domain_names `

Reply:
Filed https://msdata.visualstudio.com/Vienna/_workitems/edit/1407799

Reply:
Please add a TODO comment to clean this up that refers to that work item.